day-1/jordan-solution.js

```diff
@@ -0,0 +1,13 @@
+var sum1000 = function() {
```

Comment:
sum1000 is an awkward name. are you summing 1000 of everything?

Maybe sumMultiples3and5To(var limit) would make more sense?